src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMMA = ","; // file data separator
```

Comment:
краще старатись використовувати такй неймінг, щоб коменти не були потрібні
```suggestion
    private static final String DATA_SEPARATOR = ","; // file data separator
```